lib/membrane_element_tee/parallel.ex

```diff
@@ -1,17 +1,16 @@
 defmodule Membrane.Element.Tee.Parallel do
-  use Membrane.Filter
-
   @moduledoc """
-  Element for forwarding packets to two or more outputs
-
-  There is one input pad `:input` and output pad:
+  Element for forwarding packets to multiple outputs.
 
-  * `:output` - is a dynamic pad which is available on request and works in pull mode
+  The speed of processing is limited by the slowest consuming output.
```

Comment:
Nit: processing speed sounds more natural than the speed of processing. And uses fewer letters :wink: 
```suggestion
  The processing speed is limited by the slowest consuming output.
```